<b> Arithmetic operations

In [5]:
#addition
2+3+9

14

In [6]:
#subtraction
99-24

75

In [7]:
#multiplication
2.33*4

9.32

In [8]:
#division
99/3

33.0

In [10]:
#integer division
33//5

6

In [11]:
#Quotient
33 % 5 

3

In [12]:
#Exponential
3** 4

81

### Multistep operations

A grocery store sells a bag of ice for $1.25 and makes a 20% profit. If it sells 500 bags of ice, how much total profit does it make?

We can list out the information provided and gradually convert the word problem into a mathematical expression that can be evaluated using Python.

Cost of ice bag ($) = 1.25

Profit margin = 20% = .2

Profit per bag ($) = profit margin * cost of ice bag = .2 * 1.25

No. of bags = 500

Total profit = no. of bags * profit per bag = 500 * (.2 * 1.25)

In [13]:
500 * (0.2*1.25)

125.0

Thus, the grocery store makes a total profit of $125. While this is a reasonable way to solve a problem, it's not entirely clear by looking at the code cell what the numbers represent. We can give names to each of the numbers by creating Python variables.

In [14]:
cost_of_ice_bag = 1.25
profit_margin = .2
number_of_bags = 500

In [15]:
profit_margin

0.2

In [16]:
profit_per_bag = cost_of_ice_bag*profit_margin

In [17]:
profit_per_bag

0.25

In [19]:
total_profit = number_of_bags*profit_per_bag

In [20]:
print("The grocery store makes a total profit of $", total_profit)

The grocery store makes a total profit of $ 125.0


Evaluating conditions using Logical operators

In [21]:
my_favorite_number = 1
my_least_favorite_number = 5
a_neutral_number = 3

In [22]:
#equality_check
my_favorite_number == 1

True

In [23]:
# Not equal check - True
my_favorite_number != a_neutral_number

True

In [24]:
# Greater than check - True
my_least_favorite_number > a_neutral_number

True

In [25]:
# Greater than check - False
my_favorite_number > my_least_favorite_number

False

In [26]:
# Less than check - True
my_favorite_number < 10

True

In [27]:
# Greater than or equal check - True
my_favorite_number >= 1

True

In [28]:
# Less than or equal check - False
my_favorite_number + a_neutral_number <= 3

False

In [29]:
cost_of_ice_bag = 1.25
is_ice_bag_expensive = cost_of_ice_bag >= 10
print("Is the ice bag expensive?", is_ice_bag_expensive)

Is the ice bag expensive? False


EXERCISE: A travel company wants to fly a plane to the Bahamas. Flying the plane costs 5000 dollars. So far, 29 people have signed up for the trip. If the company charges 200 dollars per ticket, what is the profit made by the company? Create variables for each numeric quantity and use appropriate arithmetic operations.

In [30]:
plane_cost = 5000
num_people = 29
ticket_fare = 200

In [31]:
income_from_ticket = num_people*ticket_fare

In [32]:
profit = income_from_ticket - plane_cost

In [33]:
print("The profit gained by the travel company is $", profit)

The profit gained by the travel company is $ 800
